augur/tasks/start_tasks.py

```diff
@@ -347,4 +354,28 @@ def augur_collection_monitor():
             start_facade_clone_update(session,max_repo=5,days=30)
 
 
+@celery.task
+def augur_collection_update_weights():
+
+    from augur.tasks.init.celery_app import engine
+
+    logger = logging.getLogger(augur_collection_update_weights.__name__)
+
+    logger.info("Updating stale collection weights")
+
+    with DatabaseSession(logger,engine) as session:
+
+        core_weight_update_repos = session.query(CollectionStatus).filter(CollectionStatus.core_weight != None).all()
+
```

Comment:
@IsaacMilarky I think we should add filters to this query so that it only returns the repos that can be scheduled (one's that have a collection date older than 30 days). This will make sure that we aren't calculating the repo weight for not reason and it ensures we don't have to deal with the number of days being negative.

This creates another issue though. If we don't update the weight until it is available to collect then when 30 days comes around it will have the same weight as when it was last collected. My first thought it to simply have this task make two queries. One for repos that are older than 30 days, and one for repos that are older than 28 days. For the older than 30 days repos we would keep the logic the same, but for the older than 28 days repo we could calculate the weight like it is 1 hour old. This would mean when 30 days comes around all the repos weights would be set like they are 1 hour old rather than the same weight they had when they were last collected. 

This may be too much for a pr comment. Let me know if you would like to meet again to discuss it

Reply:
@IsaacMilarky / @ABrain7710 / @Ulincsys : What if, as I think I read suggested earlier, when we get to the end of any new repos, we just say "work from less recently collected to most recently collected, plus or minus the other weighting factors"? 

I may be oversimplifying .. conceptually thought this seems somewhat logical. Effectively, collection becomes continuous. 

You are also free to ignore this comment entirely and carry on. :)